ome/provider.go

```diff
@@ -219,6 +219,7 @@ func (p *omeProvider) DataSources(ctx context.Context) []func() datasource.DataS
 		NewVlanNetworkDataSource,
```

Comment:
```
# Code review report
```

**Improvement 1: Consistent Code Formatting**
The provided code has inconsistent indentation, which can lead to issues with code readability. It's essential to maintain a consistent coding style throughout the codebase. Terraform's standards recommend using the Gofmt tool to ensure consistent formatting.

Code snippet for reference:
```go
func (p *omeProvider) DataSources(ctx context.Context) []func() datasource.DataSource {
    return []func() datasource.DataSource{
        // ...
        NewAppCertDataSource,
    }
}
```

**Improvement 2: Duplicate Error Handling**
The NewAppCertDataSource function is missing error handling. It's crucial to implement proper error handling in Terraform providers. Consider using the `err` argument to handle errors and return meaningful error messages.

Code snippet for reference:
```go
func NewAppCertDataSourcestring, error) (interface{}, error) {
    // ...
    if err != nil {
        return nil, err
    }
    // ...
}
```

**Improvement 3: Potential Dead Code**
The removed code lines `-219,6 +219,7` suggest that the code might be dead code. It's essential to remove any unused code to maintain code cleanliness and reduce dead code. Review the codebase to ensure that the removed code is not used elsewhere.

These improvements will help maintain a high-quality, readable, and maintainable codebase that aligns with Hasicorp terraform standards.